recipes/framel/build.sh

```diff
@@ -1,19 +0,0 @@
-#!/bin/bash
```

Comment:
could you please revert those files which you deleted?

Reply:
I reverted the files.